release.py

```diff
@@ -71,10 +71,11 @@ def create_source_tarball():
     run_step('python', 'setup.py', 'sdist')
 
 def create_python_wheel():
-    run_step('python', 'setup.py', 'sdist', 'bdist_wheel')
+    run_step('python', 'setup.py', 'bdist_wheel')
```

Comment:
Good question. Nope -- there is still the `create_source_tarball()` function that calls `python setup.py sdist`.

In the call to twine, it uses `twine upload dist/*`, meaning, upload any distribution file, which will include the `*.tar.gz` and `*.whl` files.

I've gone ahead and merged the two create_* functions into one `create_distribution_files()` for clarity.